packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/Title.tsx

```diff
@@ -72,10 +80,16 @@ export const Default = (props: TitleProps): JSX.Element => {
   }
 
   return (
-    <ComponentContent styles={props.params.styles}>
-      <Link field={link}>
-        <Text field={text} />
-      </Link>
+    <ComponentContent styles={props.params.styles.trimEnd()}>
```

Comment:
In general, I'm wondering - is it possible to trim these strings on the server? For me it looks strange that you need to trim values everywhere, and it means that customer will need to do the same everywhere